cmd/sidecars/kafka.go

```diff
@@ -12,9 +12,9 @@ import (
 
 // KafkaConfig configurations used to create the KafkaSidecar
 type KafkaConfig struct {
```

Comment:
You also need to add `KafkaInsprAddr` in this struct. For most times it'll probably be `http://localhost` but the user can change it if he wants to

Reply:
@franciscobonand mate the `KafkaInsprAddr` is already on the struct. I will add the default value in case it comes empty.

Reply:
Am I going insane? Damn, my bad

Reply:
Added the default value but will leave the closing of this comment to you, since you might have some point that i didn't completely understood.